{
    "error": "HTTPSConnectionPool(host='bugzilla.andrew.cmu.edu', port=443): Max retries exceeded with url: /cgi-bin/cvsweb.cgi/src/sasl/plugins/digestmd5.c.diff?r1=1.170&amp;amp;r2=1.171 (Caused by NewConnectionError('<urllib3.connection.VerifiedHTTPSConnection object at 0x7f01d40a3400>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
}